### DISCUSSION TITLE
Auth-User not found in illustrated-error-layout

### DISCUSSION THREAD

**Discussion by Assimilationstheorie** [laravel/framework#39132] [Q&A]
- Laravel Version: 8.63.0
- PHP Version: 8.0.11
- Database Driver & Version: MariaDB 16

### Description:
I want to include different redirect buttons for @auth and @guest on error pages. The user who logs in via Jetstream and gets a 404 should be redirected to the dashboard. The @guest, of course, should not.

When I change the layout of errors to "illustrated-layout" & try to work with @auth and @guest there, it always shows me the div of @guest.

`dd(auth()->user())` returned `null`.

**Comment by bmpf** [comment]
Same issue here. 
Auth is null when user visits a invalid route even if its logged in.

Laravel 8.75.0 
PHP v7.3.2

**Comment by bmpf** [comment]
this seems to fix the problem, Last line in routes/web.php 

```
Route::fallback(function(){
    return response()->view('errors.404' , [] , 404);
});
```
